ios/fastlane/Fastfile

```diff
@@ -15,184 +15,27 @@
 
 default_platform(:ios)
 
-# Force Github workflows to use a local keychain for match to use
-before_all do
-  create_keychain(
-    name: "tech_at_safepaths_keychain",
-    password: "meow",
-    default_keychain: true,
-    unlock: true,
-    timeout: 3600,
-    lock_when_sleeps: false
-  )
-end
-
 platform :ios do
-    BT_APP_ID = "org.pathcheck.bt"
-    GPS_APP_ID = "org.pathcheck.covid-safepaths"
-    GPS_PROFILE = "match AppStore org.pathcheck.covid-safepaths"
-    BT_PROFILE = "match AppStore org.pathcheck.bt"
-    DEBUG_PROFILE = "match Develop org.pathcheck.covid-safepaths"
-    ADHOC_PROFILE = "match AdHoc org.pathcheck.covid-safepaths"
-    BT_STAGING_SCHEME = "BT_Staging"
-    GPS_STAGING_SCHEME = "GPS_Staging"
-    TEAM = "79Z8HUPGC3"
-    BT_SCHEME = "BT_Production"
-    GPS_SCHEME = "GPS_Production"
-    WORKSPACE = "COVIDSafePaths.xcworkspace"
-    PROJECT = "COVIDSafePaths.xcodeproj"
-    JS_PACKAGE = load_json(json_path: "../package.json")
-
-    lane :sync_local_certs do
-      match(
-        type: 'appstore',
-        readonly: true,
-        keychain_name: "tech_at_safepaths_keychain",
-        keychain_password: "meow",
-        git_basic_authorization: ENV["secrets.MATCH_GIT_BASIC_AUTH"]
-      )
-    end
-
-    desc "Push a new GPS Staging build for automation"
-    lane :staging do
-
-      signing_bundle_details = {
-        :BUNDLE_IDENTIFIER => GPS_APP_ID,
-        :PROVISIONING_PROFILE_SPECIFIER => ADHOC_PROFILE,
-        :DEVELOPMENT_TEAM => TEAM
-      }
-
-      match(
-        type: 'adhoc',
-        readonly: false,
-        keychain_name: "tech_at_safepaths_keychain",
-        keychain_password: "meow",
-        git_basic_authorization: ENV["secrets.MATCH_GIT_BASIC_AUTH"]
-      )
-
-      build_ios_app(
-        workspace: WORKSPACE,
-        scheme: GPS_STAGING_SCHEME,
-        configuration: "Staging",
-        export_method: "ad-hoc",
-        silent: true,
-        clean: true,
-        output_name: "COVIDSafePaths-staging.ipa",
+  lane :deploy_to_appcenter do |options|
+    # Push to AppCenter
```

Comment:
what do you think about removing this comment? The code seems self documenting enough here?